tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/OffloadIndexEntry.java

```diff
@@ -33,7 +33,7 @@
     /**
      * Get the entryId that this entry contains.
      */
-    long getEntryId();
+    long getFirstEntryId();
```

Comment:
Why change it to getFirstEntryId? The index always point to one entryId right?

Reply:
No, the index point to a block, I think the former name has some misleading.